src/global.h

```diff
@@ -73,7 +73,7 @@ LED bar:      lbr
 
 // version and application name (use version from qt prject file)
 #undef VERSION
-#define VERSION  APP_VERSION
+#define VERSION  GetDisplayVersion ( APP_VERSION )
```

Comment:
`GetDisplayVersion ( APP_VERSION )` used where `APP_VERSION` is currently referenced by this define, in existing code.  `APP_VERSION` wasn't referenced anywhere directly except here before.